packages/m2m-gateway/test/api/tenants/getTenants.test.ts

```diff
@@ -0,0 +1,110 @@
+import { describe, it, expect, vi } from "vitest";
+import { generateToken } from "pagopa-interop-commons-test";
+import { AuthRole, authRole } from "pagopa-interop-commons";
+import request from "supertest";
+import { m2mGatewayApi } from "pagopa-interop-api-clients";
+import { generateId } from "pagopa-interop-models";
+import { api, mockTenantService } from "../../vitest.api.setup.js";
+import { appBasePath } from "../../../src/config/appBasePath.js";
+
+describe("GET /tenants route test", () => {
+  const mockResponse: m2mGatewayApi.Tenants = {
```

Comment:
I would move this into a mock function to reuse in other tests, like done in #1848  or https://github.com/pagopa/interop-be-monorepo/pull/1814/files#diff-46202efca589d926941e26651bcca8de5e7a507a5453f43c9625b2d91d96d96fR13 and also use `generateMock` for random data, since it's a new service that requires new mocks (we don't have API mocks) let's do them properly from the beginning :)